web/packages/teleport/src/Integrations/IntegrationList.tsx

```diff
@@ -50,7 +51,7 @@ type Props<IntegrationLike> = {
   onDeletePlugin?(p: Plugin): void;
   integrationOps?: {
     onDeleteIntegration(i: Integration): void;
-    onEditIntegration(i: Integration): void;
+    onEditIntegration(i: Integration | IntegrationGitHub): void;
```

Comment:
Why is `| IntegrationGitHub` needed here? `IntegrationGitHub` is already a member of `Integration`.